examples/rapids/01-rapids-single-gpu.ipynb

```diff
@@ -0,0 +1,420 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "taxi_test = cudf.read_csv(\n",
+    "    \"https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-02.csv\",\n",
+    "    parse_dates=[\"tpep_pickup_datetime\", \"tpep_dropoff_datetime\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before creating predictions on this new dataset, it has to be transformed in exactly the way that the original training data were prepared. Thankfully you have already wrapped that transformation logic in a function!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "taxi_test = prep_df(taxi_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`cuml` comes with many functions for calculating metrics that describe how well a model's predictions match the actual values. This tutorial uses the `roc_auc_score()` to evaluate the model. This metric measures the area under the receiver operating characteristic curve. Values closer to 1.0 are desirable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_test = taxi_test.drop(columns=[\"target\"])\n",
+    "y_test = taxi_test[\"target\"]\n",
+    "\n",
+    "preds = rfc.predict_proba(X_test)[1]\n",
+    "roc_auc_score(y_test, preds)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Graph the ROC Curve\n",
+    "\n",
+    "Finally, let's look at the ROC curve. `cuml` does not have a ROC curve function, so we convert the target column and predictions to `numpy` arrays and use the `sklearn` `roc_curve` function. Plotting is done by the `seaborne` library."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fpr, tpr, _ = roc_curve(y_test.to_array(), preds.to_array())\n",
+    "\n",
+    "sns.set(font_scale=2)\n",
+    "fig = plt.figure(figsize=(8, 8))\n",
+    "plt.plot([0, 1], [0, 1], color=\"navy\", linestyle=\"--\")\n",
+    "plt.plot(fpr, tpr, color=\"red\")\n",
+    "plt.legend([\"Random chance\", \"ROC curve\"])\n",
+    "plt.xlabel(\"False positive rate\")\n",
+    "plt.ylabel(\"True positive rate\")\n",
+    "plt.xlim([0, 1])\n",
+    "plt.ylim([0, 1])\n",
+    "plt.fill_between(fpr, tpr, color=\"yellow\", alpha=0.1)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The graph shows that our results were only moderately better than random chance. It is possible that tuning hyperparameters and giving the model additional features and training data will improve this outcome."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
```

Comment:
I would punch up the conclusion a bit. Maybe instead of bullets make into a paragraph like

> As you can see, RAPIDS is great for making models like random forest train faster with a GPU. Since it mimics pandas, you can convert existing code to RAPIDS for an immediate performance boost. These techniques are especially helpful for situations like this NYC taxi dataset where the data is potentially quite large. At some point however, even one GPU may not be powerful enough for your problem, and thus you'll need to use multiple GPUs at once. To do so, check out our example on using [Rapids with a GPU Cluster](./02-rapids-gpu-cluster.ipynb)!